ctags/Makefile.am

```diff
@@ -7,6 +7,10 @@ AM_CFLAGS = \
 	$(GTK_CFLAGS) \
 	@LIBGEANY_CFLAGS@
 
+if MINGW
+AM_CFLAGS += -DUSE_SYSTEM_STRNLEN=1
```

Comment:
My knowledge of automake is very limited, so I'll better ask: Is this a correct place to define this macro? Or can it be somehow detected automatically by configure or somewhere else?